lib/ae_page_objects/rails/application_router.rb

```diff
@@ -112,6 +112,8 @@ def normalize_url(url)
           ActionDispatch::Journey::Router::Utils.normalize_path(url) unless url =~ %r{://}
         end
       end
+
+      class Rails5 < Rails4; end
```

Comment:
@jonkessler 
I was thinking about that but failed to make a good name :(
any suggestion for naming?